src/resources/Pipelines/ResultRankings/ResultRankingsInterfaces.ts

```diff
@@ -126,5 +132,18 @@ export interface ListResultRankingParams {
     perPage?: number;
     organizationId?: string;
     associatedGroups?: Array<string | null>;
+
+    /**
+     * @deprecated `ruleTypes` should be preferred over _kind_.
```

Comment:
What about the places where the Search API requires one of the `ResultRankingKind` enum values? We cannot deprecate a type that is the only valid option 🤔

Reply:
So we do indeed use the **ResultRankingKind** elsewhere in the code but I'm not deprecating that type, only the `kind` property from the ListResultRanking endpoint. If that can be problematic, I can remove the deprecated annotations. Otherwise, I'll like to make it clear the we should use the `ruleTypes` instead.